core/commands/log.py

```diff
@@ -21,12 +21,10 @@ class LogMixin(object):
     """
 
     def run(self, *args, file_path=None, branch=None):
-        self._file_path = file_path
-        self._branch = branch
-        sublime.set_timeout_async(self.run_async)
+        sublime.set_timeout_async(lambda: self.run_async(file_path, branch), 0)
 
```

Comment:
I have also noticed that reset commands are not working as they should.  Thanks for testing